config/locales/event.en.yml

```diff
@@ -46,7 +46,7 @@ en:
     mark_as_ready_for_vetting: Mark as ready for vetting
     mark_as_ready_for_vetting__desc: Case is ready for vetting
     mark_as_vetting_in_progress: Mark as vetting in progress
-    mark_as_vetting_in_progress__desc: Vetting in progress
+    mark_as_vetting_in_progress__desc: Vetting is in progress
```

Comment:
If this didn't break the tests, means we don't have tests around this wording :) 

Do we need any? Do similar bits of the app have tests on the words displayed?

Generally speaking, we'd like to see one change in the app and one change in at least one spec corresponding to it. But this is never going to break the app, so I'd tend to be guided by how this sort of thing is tested presently.

Reply:
@Polsonby,  generally I agree with your principal,
But for this case, this is the wording change in yml which is the root source of any wording , There has been a test for checking whether the wording displayed on UI match the wording from the yml--- which make sense and is enough,  there is no need to define another source of bunch of wording (duplication of words in yml) for against the wording in yml